config/webpack/webpack.config.js

```diff
@@ -154,13 +144,15 @@ const makeWebpackConfig = ({
       optimization: {
         nodeEnv: process.env.NODE_ENV,
         minimize: isProductionBuild,
+        minimizer: [new TerserPlugin({ extractComments: false })],
```

Comment:
GIven their api is a little misleading, worth adding a comment that this is to preserve inline license comments?